lib/screens/login/login.dart

```diff
@@ -24,8 +31,15 @@ class LoginScreen extends StatelessWidget {
         listener: (context, state) async {
           if (state is LoginSuccess) {
             await Future.delayed(Duration(seconds: 1));
-            // ignore: todo
-            //TODO: Navigator->Home()
+            // Redirect to Home
+            //Navigator.push(context, MaterialPageRoute(builder: (context) => HomeScreen()));
+            Navigator.pushReplacement(
+                context,
+                MaterialPageRoute(
+                    builder: (_) => BlocProvider(
+                          create: (_) => ProfileCubit(FakeHomeRepository()),
+                          child: HomeScreen(),
+                        )));
```

Comment:
since HomeScreen is yet to be completed, don't do this. replace with a TODO. It will be done when homescreen is ready.